src/NUnitEngine/Addins/nunit-v2-result-writer/NUnit2XmlResultWriter.cs

```diff
@@ -216,7 +216,7 @@ private void StartTestElement(XmlNode result)
             {
                 var suiteType = result.GetAttribute("type");
                 xmlWriter.WriteStartElement("test-suite");
-                xmlWriter.WriteAttributeString("type", suiteType);
+                xmlWriter.WriteAttributeString("type", suiteType == "ParameterizedMethod" ? "ParameterizedTest" : suiteType);
                 string nameAttr = suiteType == "Assembly" || suiteType == "Project" ? "fullname" : "name";
```

Comment:
if we're taking the trouble to fix this, maybe someone should look at other values we generate for the attribute. However, it doesn't seem like a high priority so I'd be OK with merging as is too.